src/urllib3/_collections.py

```diff
@@ -413,6 +440,15 @@ def itermerged(self) -> Iterator[Tuple[str, str]]:
             val = self._container[key.lower()]
             yield val[0], ", ".join(val[1:])
 
+    def iter_raw(self) -> Iterator[Tuple[str, bool, List[str]]]:
```

Comment:
I don't think we need this method if we choose to simplify the implementation as given above.